File: CPP/Kode Program/Items.cpp
#ifndef ITEMS_CPP
#define ITEMS_CPP

#include <iostream>
#include <string>

using namespace std;

class Items
{
    private:
        string Id;
        string Nama;
        string Kategori;
        int HargaProduk;
    
    public:
        // Constructor Default
        Items() {}

        // Setter dan Getter ID
        void setId(string id1)
        {
            Id = id1;
        }

        string getId()
        {
            return Id;
        }

        // Setter dan Getter Nama
        void setNama(string nama1)
        {
            Nama = nama1;
        }

        string getNama()
        {
            return Nama;
        }

        // Setter dan Getter Kategori
        void setKategori(string kategori1)
        {
            Kategori = kategori1;
        }

        string getKategori()
        {
            return Kategori;
        }

        // Setter dan Getter HargaProduk
        void setHargaProduk(int harga)
        {
            HargaProduk = harga;
        }

        int getHargaProduk()
        {
            return HargaProduk;
        }

        ~Items()
        {

        }
};

#endif  // ITEMS_CPP

File: Python/Kode Program/Items.py
class Items:
    def __init__(self, Id="", Nama="", Kategori="", HargaProduk=0):
        self.__Id = Id
        self.__Nama = Nama
        self.__Kategori = Kategori
        self.__HargaProduk = HargaProduk

    # Setter dan Getter ID
    def set_id(self, id1):
        self.__Id = id1

    def get_id(self):
        return self.__Id

    # Setter dan Getter Nama
    def set_nama(self, nama1):
        self.__Nama = nama1

    def get_nama(self):
        return self.__Nama

    # Setter dan Getter Kategori
    def set_kategori(self, kategori1):
        self.__Kategori = kategori1

    def get_kategori(self):
        return self.__Kategori

    # Setter dan Getter HargaProduk
    def set_harga_produk(self, harga):
        self.__HargaProduk = harga

    def get_harga_produk(self):
        return self.__HargaProduk
